website/src/components/ComparisonTable.tsx

```diff
@@ -43,7 +43,7 @@ const workspaceRows: Comparison[] = [
 		feature: 'Repo / folder structure',
 		support: {
 			moon: 'loose',
-			nx: 'strict',
+			nx: 'loose',
```

Comment:
This row now accurately provides no distinction between the tools so may ultimately want to remove it. Nx does not care about your folder structure, you can organize things however you want.